pkg/api/router/router.go

```diff
@@ -11,7 +11,7 @@ import (
 func Router(services *services.Services) (http.Handler, error) {
 	mux := services.APIServer
 
-	agents := handlers.NewAgentHandler(services.GPTClient, services.Invoker, services.ServerURL)
+	agents := handlers.NewAgentHandler(services.GPTClient, services.Invoker, services.ServerURL, services.StorageClient)
```

Comment:
The `api.Context` object passed to each of these handlers already has a client. Can we just use that one instead?

Reply:
Good call! addressed